examples/bt-get2.cpp

```diff
@@ -85,7 +85,7 @@ int main(int argc, char const* argv[])
 		, std::istream_iterator<char>()};
 
 	lt::error_code ec;
-	lt::add_torrent_params atp = lt::read_resume_data(&buf[0], buf.size(), ec);
+	lt::add_torrent_params atp = lt::read_resume_data(&buf[0], int(buf.size()), ec);
```

Comment:
Strange choice... It is slightly different from C-style cast (only syntactically AFAIK). In fact it is exactly the same "uncontrolled" cast. But, if @arvidn permits such a cast, I'm not going to go into it.